### PR TITLE
Settings: Scaffolding

### LINKED ISSUE #57: [Frontend] New Settings Page: Scaffolding

### PR DESCRIPTION
**Pull request by mxosman**
## Description of the change

Updates the structure of the Settings page to the new design. Nothing new has been added - it's still the settings page with Account Settings and Uploaded Files components - I mostly restructured the files and components to reorganize and updated the styling and render logic. This is to prepare for the new Metrics Configuration and Agency Settings components.

https://user-images.githubusercontent.com/59492998/194144104-416a86aa-f865-4b15-8198-4106448ec743.mov

## Related issues

Closes #57

### Development

**This box MUST be checked by the submitter prior to merging**:
- [x] **Double- and triple-checked that there is no Personally Identifiable Information (PII) being mistakenly added in this pull request**

These boxes should be checked by the submitter prior to merging:
- [ ] Tests have been written to cover the code changed/added as part of this pull request

### Code review

These boxes should be checked by reviewers prior to merging:

- [ ] This pull request has a descriptive title and information useful to a reviewer
- [ ] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [ ] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

**Comment by mxosman** [comment]
@lilidworkin - RE: setting up a feature branch - does this look right? I thought I'd be able to add a description to the feature branch itself, but since there are no diffs - it doesn't open a PR.

File: publisher/src/components/Settings/AccountSettings.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+import { debounce as _debounce } from "lodash";
+import React, { useRef } from "react";
+
+import { useStore } from "../../stores";
+import { TextInput, Title, TitleWrapper } from "../Forms";
+import { InputWrapper, SettingsFormPanel } from ".";
+
+export const AccountSettings = () => {
+  const { userStore } = useStore();
+  const [email, setEmail] = React.useState<string>(userStore?.email || "");
```
Comment (terryttsai) [inline publisher/src/components/Settings/AccountSettings.tsx:27]: I would generally prefer to use `??` over `||` in these cases, sometimes I forget we have `??`!

**Review by terryttsai — APPROVED** [review]

File: publisher/src/components/Settings/AccountSettings.tsx (hunk 1 above)
Comment (mxosman) [reply publisher/src/components/Settings/AccountSettings.tsx:27]: Preference noted!

**Comment by lilidworkin** [comment]
Yea @mxosman  this looks great! So the idea is that you shouldn't commit anything to this branch directly, it should just be a holding pod for other PRs that get merged into it. Can you rename it to Settings Feature Branch just to be clear? Thanks!

**Pull request merged**